extensions/wikia/Insights/scripts/InsightsListTracking.js

```diff
@@ -1,18 +1,14 @@
-define(
-	'ext.wikia.Insights.LoopTracking',
-	[
-		'wikia.tracker'
-	],
+require(['wikia.tracker'],
 	function (tracker) {
 		'use strict';
 
 		var insightType;
 
 		function onLinkClick() {
```

Comment:
Actually if we are adding both files (this and ```InsightsIndexPageTracking```) on Special Insights, maybe we can combine them if we hava the same functions in both